ports/servo/browser.rs

```diff
@@ -328,58 +356,54 @@ fn platform_get_selected_devices(devices: Vec<String>, sender: IpcSender<Option<
         match tinyfiledialogs::list_dialog("Choose a device", &["Id", "Name"], dialog_rows) {
             Some(device) => {
                 // The device string format will be "Address|Name". We need the first part of it.
-                let address = device.split("|").next().map(|s| s.to_string());
-                let _ = sender.send(address);
+                device.split("|").next().map(|s| s.to_string())
             },
             None => {
-                let _ = sender.send(None);
+                None
             }
         }
-    }).expect("Thread spawning failed");
+    }).unwrap().join().unwrap()
```

Comment:
s/unwrap()/expect("Thread spawning failed")/

Reply:
done for both...